### PR TITLE
Deprecate omnibus-supermarket as a separate repo.

### PR DESCRIPTION
**Pull request by robbkidd**
Find its continued existance in links provided in README

deprecate

### CODE REVIEW

**Comment by robbkidd** [comment]
@nellshamrell @nathenharvey @schisamo @ksubrama

**Comment by nellshamrell** [comment]
EXTERMINATE!!!
Er, I'm good to :shipit:, though I'd recommend leaving a note in the README with a link to the new consolidated project

File: README.md [hunk 1]
```diff
@@ -1,116 +1,18 @@
...
 
-This project creates full-stack platform-specific packages for
-[Chef Supermarket](https://supermarket.getchef.com).
+DO NOT SUBMIT PATCHES OR CONTRIBUTE TO THIS REPOSITORY.
 
-Issues for this project can be opened and tracked from the
-[Supermarket GitHub Issues page](https://github.com/opscode/supermarket/issues).
+The omnibus project definition for Supermarket used to live in this repo. It has
+been migrated into [Supermarket's primary git repository](https://github.com/chef/supermarket)
+under [an omnibus directory](https://github.com/chef/supermarket/tree/master/omnibus).
```
Comment (robbkidd) [inline README.md]: @nellshamrell Buried in the screenfuls of red, the README contains links to consolidation repo and omnibus directory within it.

Comment (nellshamrell) [inline README.md]: Aha!  Missed that!

**Comment by nellshamrell** [comment]
:rocket: :cake:

**Pull request merged**